docs/en/serving.md

```diff
@@ -34,6 +34,19 @@ bash workspace/service_docker_up.sh
 
 </details>
 
+<details open>
+<summary><b>7B with INT4 weight only quantization</b></summary>
+
+```shell
+python3 -m lmdeploy.serve.turbomind.deploy llama2 /path/to/llama-2-7b-chat-hf \
+    --model_format awq \
+    --group_size 128 \
+    --quant_path /path/to/awq-quant-weight.pt
```

Comment:
how to get `awq-quant-weight.pt`?

Reply:
https://github.com/mit-han-lab/llm-awq#usage